eng/pipelines/templates/steps/use-node-test-version.yml

```diff
@@ -42,7 +44,6 @@ steps:
         # pnpm v6 replaces '/' in package names with '+' to reduce nesting directory in virtual store so we need to
         # change it back
         $packageAtVersion = $packageAtVersion.Replace("+", "/")
-
```

Comment:
I have reverted this new line change and it's showing up correctly on view file of latest commit.